### PR TITLE
Add CHMOD 

### LINKED ISSUE #17: What to do with file permissions?
What about `os.chmod` and `pathlib.Path.chmod`?

### PR DESCRIPTION
**Pull request by iam-py-test**

### CODE REVIEW

**Comment by iam-py-test** [comment]
Maybe we should add an option overwrite the functions with blank functions to prevent errors

**Comment by iam-py-test** [comment]
@donno2048 for some reason, it keeps giving me this error: ValueError: too many values to unpack (expected 2)
Do you know how to fix it?

**Comment by donno2048** [comment]
> @donno2048 for some reason, it keeps giving me this error: ValueError: too many values to unpack (expected 2)
> Do you know how to fix it?

Fixed

File: ref/__init__.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 """To use this module just use the main function at the top of your code"""
 from types import ModuleType
 import importlib
-__protectfiles, __protectdirs, _lockperms = None, None,None
+__protectfiles, __protectdirs, __lockperms = None, None, None
```
Comment (iam-py-test) [inline ref/__init__.py:4]: This seems to fix the problem. 
Sorry for the typo

**Comment by iam-py-test** [comment]
Is this look ok or are there other way to use chmod?

File: ref/__init__.py (hunk 1 above)
Comment (donno2048) [reply ref/__init__.py:4]: This isn't the mistake, I fixed this too, using `_` and not `__` make this variable just protected and not hidden

**Comment by donno2048** [comment]
> Is this look ok or are there other way to use chmod?

I don't think there are other ways to do so (other than `subprocess.call(['chmod', ...])` and other things we had already eliminated)

**Comment by iam-py-test** [comment]
Is this ready to merge then?

**Comment by donno2048** [comment]
We need to update the documentation

**Comment by iam-py-test** [comment]
> We need to update the documentation

Does everything look ready to merge?

**Comment by donno2048** [comment]
Yes

**Pull request merged**